force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -287,9 +359,16 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Decimal countMoney() {
         // The list of money in the wallet, represented in dollars and cents
+        
         List<Decimal> moneyInWallet = new List<Decimal>{0.50, 10, 3.84, 24.60, 9.08, 50, 4.90}; //DO NOT CHANGE
-
-        return null; // Replace null with the variable you used to store the result
+        Decimal total = 0;
+        for( Decimal d : moneyInWallet){
+            total = total + d;
+            if( total > 40.00){
+                break;
```

Comment:
👍 Great work! You can also try checking for the value before adding to the total. Either way works!